qa/suites/rgw/iam/tasks/first.yaml

```diff
@@ -0,0 +1,7 @@
+tasks:
```

Comment:
We can delete this file as it would run all the other tests which I guess is not the motto here.